src/test/java/com/github/brandtg/PlotTest.java

```diff
@@ -35,6 +36,17 @@ public void testPlot() throws Exception {
 
     final StlDecomposition stl = new StlDecomposition(config);
     final StlResult res = stl.decompose(ts, ys);
+
+    final File output = new File("seasonal.png");
+    final File hourly = new File("stl-hourly.png");
+
+    StlPlotter.plot(res, "New Title", Hour.class, hourly);
+    StlPlotter.plot(res, output);
     StlPlotter.plot(res);
+
+    assert (hourly.exists());
```

Comment:
Can you change `assert` to TestNG's `Assert.assertTrue(...)` http://testng.org/javadocs/org/testng/Assert.html#assertTrue-boolean-